e2-studio-star-rx72n-firmware/libs/rx_crc/src/rx_dmaca.c

```
@@ -0,0 +1,314 @@
+
+/* =========================================================================
+ * Internal helpers — register access
+ *
+ * The RX72N iodefine.h exposes DMAC channels as an array:
+ *   DMAC0 … DMAC7  or  DMAC[0] … DMAC[7]  (BSP-version-dependent)
+ *
+ * We use a small inline accessor to keep channel selection uniform.
+ * ========================================================================= */
+
+/** Return a pointer to the volatile DMAC channel register block. */
+static volatile struct st_dmac0 * dmac_ch(uint8_t ch)
```

Clang (14.0.6)

[warning] parameter name 'ch' is too short, expected at least 3 characters

(readability-identifier-length)